### ISSUE TITLE
About forward module.

### ISSUE THREAD

**Issue by DonghoonPark12** [HiKapok/SSD.TensorFlow#89]
Hi author. thanks for contribution.

My question is ,,
I don't understand role of the **forward function**.

def forward_module(m, inputs, training=False):
    if isinstance(m, tf.layers.BatchNormalization) or isinstance(m, tf.layers.Dropout):
        return m.apply(inputs, training=training)
    return m.apply(inputs)

The input '**m**' is  convolution layer in **'self._conv#_block'** 
which is 'tf.layers.Conv2D'.

And,  in forward_module it check whether '**tf.layers.Conv2D**' is instance of '**tf.layers.BatchNormalization**' or ' **tf.layers.Dropout**' ?? Is it make sense??

If I misunderstand sth, I'd appreciate your correction, Thanks ahead.

**Comment by HiKapok** [comment]
@DonghunP 'tf.layers.BatchNormalization' or ' tf.layers.Dropout' is the two special case which have different behavior in training or testing
